Templates/WebHook-GitHub/index.js

```diff
@@ -1,6 +1,6 @@
-﻿module.exports = function (body, context) {
+﻿module.exports = function (context, body) {
```

Comment:
In samples, for WebHooks we've standardized on using 'data' rather than 'body' for the payload. @davidebbo likes that best.